db/seeds.rb

```diff
@@ -1,56 +1,222 @@
 # frozen_string_literal: true
 
-require 'csv'
+firestone = Building.find_or_create_by(name: 'Firestone Library', email: 'access@princeton.edu')
+lewis = Building.find_or_create_by(name: 'Lewis Library', email: 'lewislib@princeton.edu')
+Rails.logger.warn("Created #{Building.count} Buildings")
 
-Building.seed
+# Setup flipflop for feature swap - borrowed from https://github.com/voormedia/flipflop/blob/ed1fd56f2f86974004c7af819612dd1622adcf33/lib/tasks/flipflop.rake
+# For some reasone when we use the rake tasks directly, only the first one has an impact
+m = Object.new
+m.extend Flipflop::Rake::SupportMethods
 
-# File was generated using csv:users rake task
-CSV.parse(File.open(Rails.root.join('new_users.csv')), headers: true) do |row|
-  User.find_or_create_by(row.to_hash)
+# Turn Lewis features on
+m.switch_feature!('lewis_staff', 'active_record', true)
+m.switch_feature!('lewis_patrons', 'active_record', true)
+Rails.logger.warn('Turned on Lewis features')
+
+# Lewis features off
+# m.switch_feature!('lewis_staff', 'active_record', false)
```

Comment:
I'm not sure we'll remember what this commented-out code does after time passes.  Could we remove it, especially since Lewis is going live soon?

Alternatively, if we want to keep it, could we have it behind some kind of config option (maybe an environment variable)?